freezeyt/freezer.py

```diff
@@ -62,29 +63,51 @@ async def freeze_async(app, config):
     '5xx': 'error',
 }
 
+def mime_db_mimetype(mime_db: dict, url: str) -> Optional[List[str]]:
+    """Determines file MIME type from file suffix. Decisions are made
+    by mime-db rules.
+    """
+    suffix = PurePosixPath(url).suffix[1:].lower()
+
+    return mime_db.get(suffix)
 
 
-def default_get_mimetype(url: str) -> Optional[str]:
-    """Returns filetype as a string from mimetype.guess_type
+def default_mimetype(url: str) -> Optional[List[str]]:
+    """Returns file mimetype as a string from mimetype.guess_type.
+    file mimetypes are guessed from file suffix.
     """
-    file_type, encoding = guess_type(url)
-    return file_type
+    file_mimetype, encoding = guess_type(url)
+    if file_mimetype is None:
+        return None
+    else:
+        return [file_mimetype]
 
 
 def check_mimetype(
     url_path, headers,
-    default='application/octet-stream', *, get_mimetype=default_get_mimetype,
+    default='application/octet-stream', *, get_mimetype=default_mimetype,
 ):
+    """Compare mimetype sent from headers with file mimetype guessed
+    from its suffix.
+    """
     if url_path.endswith('/'):
         # Directories get saved as index.html
         url_path = 'index.html'
-    file_type = get_mimetype(url_path)
-    if not file_type:
-        file_type = default
+    file_mimetypes = get_mimetype(url_path)
+    if file_mimetypes is None:
+        file_mimetypes = [default]
+
     headers = Headers(headers)
-    mime_type, encoding = parse_options_header(headers.get('Content-Type'))
-    if file_type.lower() != mime_type.lower():
-        raise WrongMimetypeError(file_type, mime_type, url_path)
+    headers_mimetype, encoding = parse_options_header(
+        headers.get('Content-Type')
+    )
+
+    if isinstance(file_mimetypes, str):
+        raise TypeError(
+            f"get_mimetype result must not be a {type(file_mimetypes)!r}"
+        )
```

Comment:
Is good idea to use
```python
{type(file_mimetypes)!r}
```
or just
```python
string
```
is better?

Reply:
I think just `string` is best